resources/server/commands/registerCommands.ts

```diff
@@ -0,0 +1,53 @@
+import { mainLogger } from '../sv_logger';
+import { config } from '../config';
+import DbInterface from '../db/db_wrapper';
+
+interface ColumnInfo {
+  COLUMN_NAME: string;
+  DATA_TYPE: string;
+  CHARACTER_MAXIMUM_LENGTH: number;
+  IS_NULLABLE: string;
+}
+
+const npwdDebugDumpCommand = async ([src]: [number]): Promise<void> => {
+  // We require this be called from the server console.
+  if (src !== 0) return;
+
+  mainLogger.info('NPWD DEBUG DUMP STARTED, THIS WILL WRITE TO THE SV_NPWD.LOG FILE');
+  mainLogger.info('Resource Config >');
+  mainLogger.info(config);
+
+  const versionInfo = GetResourceMetadata(GetCurrentResourceName(), 'version', 0);
+  mainLogger.info(`Manifest VERSION > ${versionInfo}`);
+
+  const fxServerVersion = GetConvar('version', 'unknown');
+  mainLogger.info(`FXServer VERSION > ${fxServerVersion}`);
+
+  const activePlayerCount = GetNumPlayerIndices();
+  mainLogger.info(`Connected Player Count > ${activePlayerCount}`);
+
+  try {
+    const playerTableResults = await DbInterface.fetch<ColumnInfo[]>(
+      `SELECT COLUMN_NAME, DATA_TYPE, CHARACTER_MAXIMUM_LENGTH, IS_NULLABLE
+       FROM INFORMATION_SCHEMA.COLUMNS
+       WHERE TABLE_NAME = '${config.database.playerTable}'`,
+    );
```

Comment:
I could be wrong here but wont INFORMATION_SCHEMA contain data from other databases on the same server, unless you specify the database?

Reply:
You're probably right, I forgot about that specifier when writing this. It has yet to be tested either.